### ISSUE TITLE
Project Report

### ISSUE THREAD

**Issue by Carson-MD** [Carson-MD/4400#2]
The project report - details to follow.

**Comment by Carson-MD** [comment]
Here's a brief overview of what we will include in our final report.

ABSTRACT
-Brief summary of our work and what we accomplished.
INTRO
-A brief introduction (worth discussing that we there is now a lot of data in the world, and how aggregating that data and drawing meaningful conclusions from it difficult)
CONCEPTS OVERVIEW
-Explain the scientific concepts required to understand the work (genes, alleles, interaction between genetics and pharmacokinetics, drug concentrations in the body and the relationship between dosing and metabolism).
OBJECTIVES
-What we aimed to accomplish.
RESULTS
-What we managed to achieve.
FUTURE WORK
-What's next (expand drugs & genes in the database, search for additional relationships, refine algorithms, USER INTERFACE).
CONCLUSION
-Conclude things.
